openedx/features/course_experience/static/course_experience/js/CoursetalkReviews.js

```diff
@@ -0,0 +1,29 @@
+/*
+  This file is to enable users on the coursetalk reviews
+  page to both view and write reviews.
+ */
+
+export class CoursetalkReviews {  // eslint-disable-line import/prefer-default-export
+  constructor() {
+    const $coursetalkToggleReadWriteReviews = $('.toggle-read-write-reviews');
+    const $coursetalkScriptObject = $('.coursetalk-reviews-script');
+
+    const readSrc = '//d3q6qq2zt8nhwv.cloudfront.net/s/js/widgets/coursetalk-read-reviews.js';
```

Comment:
I don't love that this is hard-coded, but maybe that is ok.  But, I also don't like that this is duplicated here and in the script. Should it come in through the context?  @andy-armstrong: How do we typically handle this type of thing?